src/nbl/video/device_capabilities/device_features.json

```diff
@@ -544,10 +544,10 @@
         {
             "type": "bool",
             "name": "samplerYcbcrConversion",
-            "value": false,
+            "value": true,
             "expose": "DISABLE",
```

Comment:
look at the `expose` its disabled, means we don't even care about it, so leave the value as was